setup.py

```diff
@@ -71,6 +72,11 @@
     packages=find_packages(),
     namespace_packages=[],
     include_package_data=True,
+    package_data={
+        "artiq": [
+            "gateware/targets/kasli_generic.schema.json",
+        ],
```

Comment:
For some reason that I cannot remember, this doesn't work in certain situations (Python package management is incredibly shitty). The less broken way is to use ``MANIFEST.in``.